src/components/payment/context-map.model.ts

```diff
@@ -0,0 +1,24 @@
+export class ContextMap {
```

Comment:
if the ContextMap lives in the toolkit, when in management web we want to pass a new property to a component that lives in management web only, do we have to release a new version of the toolkit?
Is it possible to find a way to have this map live in management web?